CONTRIBUTING.md

```diff
@@ -1,55 +1,77 @@
-# Contributing
+# How to Contribute to Spectrum Web Components
 
-Thanks for choosing to contribute!
+First off, thanks for taking the time to contribute!
 
-The following are a set of guidelines to follow when contributing to this project.
+The following is a set of guidelines for contributing to Spectrum Web Components which is hosted in the [Adobe Spectrum Web Components](https://github.com/adobe/spectrum-web-components) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
 
-## Code Of Conduct
+## I don't want to read this whole thing I just have a question!!!
```

Comment:
Maybe rename this to just TL;DR; or something about the content more directly. This title and the next effectively says "have a question".

Reply:
Or something like "Things to know"?

Reply:
I like TL;DR!